tests/tests.py

```diff
@@ -215,19 +215,42 @@ def __init__(self):
                 # [7-8] python version A=0, B=1,..
                 # [9] A,B -- we may need more than one project
 
-                prefix = 'Z' + (re.sub("[^A-Z]", "",
-                                       getpass.getuser().upper()))[0:6] + \
-                         str(sys.version_info[0]) + \
-                         str(sys.version_info[1])
+                """ `jid` is important for avoiding concurency problems when
```

Comment:
Just trying to grok these changes. Looks like the comments above no longer reflect what the code is doing, which is not helpful :-(